pyproject.toml

```diff
@@ -27,6 +27,8 @@ python-dateutil = "^2.8.1"
 retry = "^0.9.2"
 spurplus = "^2.3.4"
 semver = "^2.13.0"
+pypandoc = "^1.6.3"
```

Comment:
It looks they should be dev dependencies. Is it right?

Reply:
No longer needed. Deleted.